connector_manager/requirements.txt

```diff
@@ -18,4 +18,5 @@ requests==2.28.1
 six==1.16.0
 tenacity==8.1.0
 toml==0.10.2
-urllib3==1.26.12
\ No newline at end of file
+urllib3==1.26.12
+gnupg==2.3.1
```

Comment:
how was this possible without? in Dockerfile there is just install requirements.txt and the whole thing is based on this module? @mre

Reply:
Maybe it was a sub-dependency of something else and thus was implicitly installed. You can check the dependency tree to find out.